datalad/support/tests/test_annexrepo.py

```diff
@@ -1134,7 +1134,8 @@ def test_annex_copy_to(origin, clone):
     repo.get("test-annex.dat")
     # now it has:
     eq_(repo.copy_to("test-annex.dat", "target"), ["test-annex.dat"])
-    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), ["test-annex.dat"])
+    # and will not be copied again since it was already copied
+    eq_(repo.copy_to(["INFO.txt", "test-annex.dat"], "target"), [])
```

Comment:
Yes, we already started to deal with that kind of stderr parsing in `_run_annex_command_json`, I think, but it starts getting messy without a proper design for that ...